aten/src/ATen/native/cuda/MiscUtils.h

```diff
@@ -89,6 +89,14 @@ struct MagmaStreamSyncGuard {
 };
 #endif
 
+static inline int cuda_int_cast (int64_t value, const char* varname) {
```

Comment:
Maybe you can templatize `magma_int_cast` to give this function? It's pretty much the same.

Reply:
The reason is that `magma_int`cast` is defined inside a `#ifdef USE_MAGMA` block. This `cuda_int_cast` doesn't have any define guard. I will clean up this later with all the MAGMA and cuda guards.